edx_sga/tests.py

```diff
@@ -1,676 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
It looks like this file still exists. Can you delete it instead of just clearing its contents?